__tests__/app.test.js

```diff
@@ -82,3 +82,39 @@ describe("GET 200 /api/articles/:article_id", () => {
       });
   });
 });
+describe("GET /api/articles", () => {
+  test("get 200 return all articles including comment count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(5);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+              comment_count: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
```

Comment:
there should also be a test to make sure they are coming back in created_at descending order